pkg/model/estimator/local/ratio_test.go

```diff
@@ -61,17 +64,17 @@ var _ = Describe("Test Ratio Unit", func() {
 			Pkg:    20,
 		}
 		nodeMetrics.AddNodeComponentsEnergy(componentsEnergies)
-		Expect(nodeMetrics.EnergyInCore.Curr()).Should(BeEquivalentTo(10))
-		Expect(nodeMetrics.EnergyInDRAM.Curr()).Should(BeEquivalentTo(3))
-		Expect(nodeMetrics.EnergyInUncore.Curr()).Should(BeEquivalentTo(2))
-		Expect(nodeMetrics.EnergyInPkg.Curr()).Should(BeEquivalentTo(20))
+		Expect(nodeMetrics.EnergyInCore.Aggr()).Should(BeEquivalentTo(10))
```

Comment:
so Curr() equals with Aggr()?

Reply:
In this case, since there is one power update, Curr and Aggr are the same. @sunya-ch